_events/2016-09-24.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Noise Hackathon"
+description: "The Noise Hackathon centers on the notion of “Noise in 3D: Data-Driven, Art-Driven & Community-Driven” while embracing both the vastness and idiosyncrasies of the world of noise. Talks, panels, and workshops will provide gateways into the complexities of noise including urban noise pollution and noise music to jump-start a full day of noise hacking. Please join us in making some noise."
+date: 2016-09-24 11:30am
+address: "45 W 18th St, New York, NY 10011"
+rsvp: "https://www.eventbrite.com/e/noise-hackathon-tickets-26751738169"
+image: "2016-09-24.gif"
+---
+
+# Schedule
+
+- 11:30 AM Doors open
+- 12:00 PM Talks
+- 01:20 PM Hacking and optional brainstorming session start
+- 02:00 PM Workshops
+- 08:00 PM Project presentations
+
+# Talks
+
+- Arline Bronzaft -
+  Less Noise Translates Into a Healthier Lifestyle
+
+- Tae Hong Park -
+  Soundmapping Our World: Measuring, Seeing, Doing
+
+- Dafna Naphtali -
+  Interactive Site-specific Composition: U-GRUVE Audio Augmented Reality
+
+- Nikita Scott -
+  Healthy Oceans: The True Cost of Underwater Noise
+
+# Workshops
+
+- Using the Citygram system and Supercollider to interact with and display NYC noise pollution data
+  
+special thanks to Tony Lim for creating the background image. 
```

Comment:
I don't think these old events matter much because we're just backfilling them for completeness but... I think you dropped a couple words: 
"We extend a special thanks to Tony Lim for creating the background image."